episodes/16-what-is-myst.Rmd

```diff
@@ -29,6 +29,12 @@ MyST is an open-source, community-driven project to improve scientific communica
 Myst is a tool that we will learn to use to build interactive webpages or pdfs. The benifit of this is more powerful publishing of content and information.
 Here's an example of a paper published using myst: [A Literature Review On AI fairness Best Practices](https://ml4sts.com/fairml-bestpractices/)
 
+## A Different Way of Building HTML
+The purpose of MyST is to have a powerful tool that builds interactive HTML pages without doing any HTML coding.
+Remember, on the first day we used GH Pages' simple rendering system to render a single md file (`index.md`) into a simple HTML page.
+MyST will do a similar job, except that it can render a more powerful HTML page with interactive button and it can render a collection of md files distributed into multiple directories within one project directory.
```

Comment:
```suggestion
MyST can also do this, except that it uses templates to render pages with a more structure, does some extra processing of the content and it can render a collection of md files distributed across multiple directories within one project directory into a cohesive site.
```